data/tech/aria/aria-rowcount_attribute.json

```diff
@@ -9,6 +9,46 @@
       "url": "https://www.w3.org/TR/wai-aria-1.1/#aria-rowcount"
     }
   ],
-  "assertions": [],
-  "date_updated": "2019-01-06"
+  "assertions": [
+    {
+      "id": "convey_row_count_all_rows",
+      "title": "convey the total row count when all rows are visible",
+      "rationale": "The user needs to know how many rows are in the whole grid / table",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "*[aria-rowcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "convey_row_count_hidden_rows",
+      "title": "convey the total row count when some rows are hidden",
+      "rationale": "The user needs to know how many rows are in the whole grid / table when some rows are hidden",
+      "strength": {
+        "sr": "MUST",
+        "vc": "NA"
+      },
+      "css_target": "*[aria-rowcount]",
+      "operation_modes": [
+        "sr/reading"
+      ]
+    },
+    {
+      "id": "do_not_calculate_unknown_row_count",
+      "title": "convey a row count when total row count is unknown",
```

Comment:
Awesome expectation, good work! I had forgotten that ARIA mentions this.

My only concern is that the expectation doesn't mention anything about "-1" being the magical value to indicate this. We could add a "notes" property to the expectation and explain this. Thoughts?

Reply:
Added note verbatim from the spec. ✔️